_includes/master/charts/calico/values.yaml

```diff
@@ -2,6 +2,16 @@
 # Note: Changing this after installation is not supported. 
 datastore: kdd
 
+# Config for etcd
+etcd:
+  # Endpoints for the etcd instances. This can be a comma separated list of endpoints.
+  endpoints: "http://<ETCD_IP>:<ETCD_PORT>"
```

Comment:
Anything that is "how the docs do it" should go into `helm.rb` if possible. So pass `--set etcd.endpoints=
http://<ETCD_IP>:<ETCD_PORT>"` from there.

In values.yaml, this should default to `nil`